WebClient/src/components/moduleControl/moduleControlWindow/moduleControlItemList/moduleControlItemListView.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * ModuleControlItemListView. View for displaying a list of modules.
+ */
+
+/* globals L */ 
+
+import './moduleControlItemList.css';
+import View from '../../../../core/view';
+import ModuleControlItemView from '../moduleControlItem/moduleControlItemView';
+
+var ModuleControlItemListView = View.extend({
+
+    onInitialize: function (opts) {
+        if (!opts) throw new Error('No arguments are provided to the View');
+        if (!opts.modules) throw new Error('modules is not provided');
```

Comment:
I think it should be 'Modules are not provided'.

Reply:
Changed to 'opts.modules property is not... '